src/local/launch-keycloak.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+SCDIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
+SCDIR=$(realpath $SCDIR)
+
+docker run --name keycloak  -p 8080:8080 -p 9000:9000 \
```

Comment:
Curious.   Why does the docker compose version require a database and this one does not?

Reply:
Keycloak will use an embedded in memory database if you don't provide jdbc credentials.

Reply:
Thank you!!!!